spec/graphql/resolvers/subscriptions_resolver_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Resolvers::SubscriptionsResolver, type: :graphql do
+  let(:query) do
+    <<~GQL
+      query {
+        subscriptions(limit: 5, planCode: "#{plan.code}") {
+          collection { id externalId plan { code } }
+          metadata { currentPage, totalCount }
+        }
+      }
+    GQL
+  end
+
+  let(:membership) { create(:membership) }
+  let(:plan) { create(:plan, organization:) }
+  let(:organization) { membership.organization }
+  let(:customer) { create(:customer, organization:) }
+
+  before do
+    customer
+  end
+
+  it 'returns a list of subscriptions' do
+    first_subcription = create(:subscription, customer:, plan:)
+    second_subcription = create(:subscription, customer:, plan:)
+    third_subcription = create(:subscription, customer:)
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `third_subcription`. Did you mean `first_subcription`? (https://rubystyle.guide#underscore-unused-vars)